content/news/2020-08-28-gsoc-ends-2020.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Google Summer of Code Wrap-up"
+date = 2020-08-28
+extra.author = "Devin Ragotzy"
++++
+
+Wow the summer has flown by, it feels like just yesterday I was learning how to rebase and what exactly it is ruma does. I exaggerate slightly, but it is a big library with lots of public API surface. I have learned more in the last few months than in two years of school. I have been able to observe and participate in a project with a community growing around it, been apart of discussions about design and best practices, given and received numerous code reviews as well as learned the process of addressing the feedback, and working from a specification. In short this has been an amazing opportunity to gain experience in all the things that are hard to obtain in a classroom.
+
+My project goal was to improve the existing macros in ruma-events-macros and ruma-api-macros. It became clear early on that this would include some major API changes and that improving the macros as they were was pointless without also moving to the new API design. While improving the durability and readability of the macro code I also rewrote entire sections to accommodate the new design.
```

Comment:
```suggestion
My project goal was to improve the existing macros in ruma-events-macros and ruma-api-macros. It became clear early on that this would include some major API changes and that improving the macros as they were was pointless without also moving to a new public API. While improving the durability and readability of the macro code I also rewrote entire sections to accommodate the new design.
```